kblocks/data/cache.py

```diff
@@ -81,7 +81,7 @@ def ret_transform(dataset):
             cache_factory(p)(dataset)
 
         return tf.data.Dataset.from_tensor_slices(paths).flat_map(
-            lambda p: transform(cache_factory(p)(dataset))
+            lambda _p: transform(cache_factory(_p)(dataset))
```

Comment:
I see the argument against using `p`, but is there an argument for `_p` over `p_`? `_p` implies to me something fundamentally different to `p` related to protection, where `p_` is what I've seen done to avoid potential kwarg conflicts e.g. [here](https://www.tensorflow.org/api_docs/python/tf/compat/v1/make_template)

Reply:
Yeah just avoiding using the variable in outer scope. Renaming to `p_` is fine.